ReadMe.md

```diff
@@ -164,7 +164,17 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 * Salma Shaik
 * Prachi Mane
 
-## FAQs
+## General FAQs.
+
+#### What is OpenSource Projects
+
+ OpenSource Projects means anybody is free to use, study and modify your project. The open source community provides a great opportunity for aspiring programmers to distinguish themselves; and by contributing to various projects, developers can improve their skills and get inspiration and support from like-minded people. But most importantly, they can prove that they can build fantastic experiences that people love. 
```

Comment:
- Project, not projects.
- an Open Source.
- remove the space before the first line.
- remove the space after the para.